cmake/FindBreathe.cmake

```diff
@@ -0,0 +1,30 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Look for an executable called breathe-apidoc
+find_program(
+  BREATHE_APIDOC_EXECUTABLE
+  NAMES breathe-apidoc
+  PATHS
+  ${BREATHE_APIDOC_ROOT} ${BREATHE_ROOT}
+  ENV BREATHE_APIDOC_ROOT ENV BREATHE_ROOT
```

Comment:
If I understand https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure correctly, the policy for `_ROOT` variables is that they match the package name exactly, including case. So here we should have `Breathe_ROOT` and `Sphinx_ROOT`. We don’t do this consistently in other places though, so I don’t request you change that. If you do, please squash directly.

Reply:
I'm just fixing it to work the way CMake's own modules work... Then we also avoid uninitialized variable warnings

Reply:
Ah okay, so that's if we want to support that CMake functionality. I have no problem adding things like `Breathe_ROOT`, but we should do that uniformly. I think currently we always do `BLAH_ROOT`